### DISCUSSION TITLE
Improve documentation for separate version tracks

### DISCUSSION THREAD

**Discussion by p-fischer** [InsertKoinIO/koin#1442] [General]
As documented in https://insert-koin.io/docs/setup/v3.2 there are different version tracks for JVM, Android & Android compose libraries now.

However, as https://github.com/InsertKoinIO/koin/issues/1431 indicates there might still be some confusion.

What confused me was that the changelog https://github.com/InsertKoinIO/koin/blob/main/CHANGELOG.md#321---2022-09-12 didn't announce there is a separate version for Android compose too.

It might help to include the latest version numbers of all tracks in the Readme file and to provide separate release notes. 
(What I find confusing is that when koin `3.2.1` was released there was no koin compose version `3.2.1`. Now koin `3.2.2` was released and there is koin compose version `3.2.1` but I can't see anything in the release notes.)

**Comment by arnaudgiuliani** [comment]
Thanks for your feedback.

The idea is to have dedicated milestones per project, like https://github.com/InsertKoinIO/koin/milestone/38

In changelog, a dedicated section is written down.

**Reply by arnaudgiuliani** [reply]
I can also add a version tag on on each new section, added in documentation. Like `(since 3.4.3)`
